src/Integration/Binding/CFamilyProjectBinder.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO.Abstractions;
+using System.Linq;
+using EnvDTE;
+using SonarLint.VisualStudio.Integration.NewConnectedMode;
+
+namespace SonarLint.VisualStudio.Integration.Binding
+{
+    internal class CFamilyProjectBinder : IProjectBinder
```

Comment:
As we discussed, solution-level language-specific test should still happen somewhere. I've put it here for now, but I'm unsure of its final destination; there is a logical duplication here, as the solution is being checked over and over again, so it should probably be checked only once, as part of the solution-level checks. In the meanwhile it's here, I'll revisit this once the other parts are integrated.

Reply:
From a practical point of view, repeating the check for each project won't cause a problem in practice.